x-pack/plugins/security_solution/public/cases/components/timeline_actions/helpers.tsx

```diff
@@ -20,7 +29,7 @@ export const createUpdateSuccessToaster = (
     id: uuid.v4(),
     color: 'success',
     iconType: 'check',
-    title: i18n.CASE_CREATED_SUCCESS_TOAST(theCase.title),
+    title: <Title>{i18n.CASE_CREATED_SUCCESS_TOAST(theCase.title)}</Title>,
```

Comment:
`theCase.title` should probably also be passed as the `title` attribute, which would be used by screen readers and should also show up on hover